dockerfiles/base/Dockerfile

```diff
@@ -26,25 +26,12 @@ RUN apt update -qqq \
 
 ENV SSL_CERT_DIR /etc/ssl/certs
 
-RUN python -m pip install git+https://github.com/${MLRUN_GITHUB_REPO}/mlrun.git@${MLRUN_PACKAGE_TAG}
+COPY ./dockerfiles/base/requirements.txt ./
 
-RUN python -m pip install urllib3==1.25.8  kubernetes==10.0.1
+RUN python -m pip install -r requirements.txt
 
-RUN python -m pip install scikit-learn
+RUN python -m pip install \
+    git+https://github.com/${MLRUN_GITHUB_REPO}/mlrun.git@${MLRUN_PACKAGE_TAG}
 
 RUN python -m pip install \
-    dask-kubernetes \
-    dask-ml \
-    dask-xgboost \
-    dask[complete] \
-    distributed
-
-RUN python -m pip install --no-cache-dir \
-    blosc==1.7.0 \
-    cloudpickle==1.1.1 \
-    dask==2.12.0 \
-    distributed==2.14.0 \
-    lz4==2.2.1 \
-    msgpack==1.0.0 \
-    tornado==6.0.4 \
-    numpy==1.18.1
+    git+https://github.com/mlrun/mlutils.git@9182bf7d0e20ca6c48d9662bf8776efae2b4f1b7
```

Comment:
what is this tag ?
Suggest to make it an `ARG` - `MLUTILS_GITHUB_TAG` and probably we want the default to be something like `development` or `master` (same for all other docker files)

Reply:
It would be great if we added a build arg for the mlutils package so that this becomes unnecessary

Reply:
`development` would be good

Reply:
Please add build arg to the dockerfile with `development` as default, I'll support that later in the makefile